woocommerce/wc-admin
# [WIP] Try/fresh data within wp data

Note: This uses unmerged code from `@wordpress/data` in https://github.com/WordPress/gutenberg/pull/8754

This PR tries out some updates proposed for `@wordpress/data` which adds `fresh-data` functionality within it. It implements the orders list from the activity panel.

![dashboard_ _woo_dash_test_site_ _woocommerce](https://user-images.githubusercontent.com/945228/43616531-a5087110-9682-11e8-851e-998075fed0ba.png)

To Test:
0.1. Set up my PR branch in your gutenberg plugin repo by either cloning [my fork](https://github.com/coderkevin/gutenberg.git) or setting it up as an alternate remote.
0.2. From the gutenberg repo, checkout my `try/fresh-data-as-data-plugin` branch.
0.3. From gutenberg, run `npm run build`.

1. Build this branch as normal.
2. Test out functionality on the orders. It should show "Loading..." and then the orders. And clicking the button should actually fulfill each order.
